
<!DOCTYPE html>
<html lang="en">
<head>
	<title> Summary: &#34;Mutators and sex in bacteria: Conflict between adaptive strategies&#34; (Tenaillon et al. 2000) </title>
	<meta name="description" content="Yoav Ram's blog on evolutionary theory and other scientific interests, and especially on the theoretical treatment of the evolution of the mutation rate" />
	<meta name="author" content="Yoav Ram">
	<meta name="keywords" content="evolution,mutation rate,mutation rates,Yoav Ram,yoavram,evolutionary theory,science,theoretical biology,mathematical biology,mathematics">
	<meta charset="UTF-8">
	<meta http-equiv="content-type" content="text/html; charset=UTF-8">
	<link rel="icon" type="image/ico" href="/static/favicon.ico">
	<link href="http://feeds.feedburner.com/yoavram/msb-recent" rel="alternate" title="Recent Articles" type="application/atom+xml">

	<!-- Le style -->
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<link href='http://fonts.googleapis.com/css?family=Droid+Sans:400,700' rel='stylesheet' type='text/css'>
	<link href="/static/theme.css" rel="stylesheet">
	<link href="/static/bootstrap.min.css" rel="stylesheet">
	<link href="/static/glyphicons.css" rel="stylesheet">
	<link href="/static/bootstrap-responsive.min.css" rel="stylesheet">
		<link href="/static/jekyll.css" rel="stylesheet">

	<style type="text/css">
	[class^="icon-"] { background-image: url("/static/glyphicons-halflings.png"); } 
	[class*="icon-white"] { background-image: url("/static/glyphicons-halflings-white.png"); } 
	[class^="glyphicon-"] { background-image: url("/static/glyphicons.png"); } 
	</style>
	<link href="/static/prettify.css" rel="stylesheet">

	<!-- Le JS -->
	<script type="text/javascript" src="/static/jquery-1.7.2.min.js"></script>
	<script type="text/javascript" src="/static/bootstrap.min.js"></script>

	<!-- google code prettifier http://code.google.com/p/google-code-prettify/ -->
	<script type="text/javascript" src="/static/prettify.js"></script>

	



<!-- MathJax -->
<script type="text/x-mathjax-config">
<!-- Online Debugger: http://www.onemathematicalcat.org/MathJaxDocumentation/TeXSyntax.htm -->
<!-- Doc: http://docs.mathjax.org/en/latest/output.html -->`
  MathJax.Hub.Config({            
			"HTML-CSS": { availableFonts: ["STIX", "TeX"], 
						  linebreaks: { automatic: true }
			}
  });
</script>
<script type="text/javascript"
  src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML">
</script>




<!-- ShareThis JS -->
<script type="text/javascript">var switchTo5x=true;</script>
<script type="text/javascript" src="http://w.sharethis.com/button/buttons.js"></script>
<script type="text/javascript">stLight.options({publisher: "{{ config.SHARETHIS_PUBLISHER }}"});</script>
<!-- ShareThis JS -->




</head>


<body onload="prettyPrint()">

	<!-- Navbar -->
	<div class="navbar navbar-fixed-top">
		<div class="navbar-inner">
			<div class="container">

				<!-- .btn-navbar is used as the toggle for collapsed navbar content -->
				<a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
				</a>

				<!-- Be sure to leave the brand out there if you want it shown -->
				<a class="brand" href='/'>The Mutation-Selection Blog</a>

				<!-- Everything you want hidden at 940px or less, place within here -->
				<div class="nav-collapse">
					<ul class="nav">
						<li class="dropdown">
							<a href="#" class="dropdown-toggle" data-toggle="dropdown" title="Tags">
								<!--Tags -->
								<i class="icon-tags"></i>
								<b class="caret"></b>
							</a>
							<ul class="dropdown-menu">
								

	<li><a href='/tag/R/'>R: <small>1</small></a></li>

	<li><a href='/tag/asexual%20populations/'>asexual populations: <small>4</small></a></li>

	<li><a href='/tag/citations/'>citations: <small>1</small></a></li>

	<li><a href='/tag/evolution%20of%20the%20mutation%20rate/'>evolution of the mutation rate: <small>1</small></a></li>

	<li><a href='/tag/latex/'>latex: <small>1</small></a></li>

	<li><a href='/tag/markdown/'>markdown: <small>6</small></a></li>

	<li><a href='/tag/mathematical%20modeling/'>mathematical modeling: <small>2</small></a></li>

	<li><a href='/tag/mean%20fitness/'>mean fitness: <small>2</small></a></li>

	<li><a href='/tag/mutation-selection%20balance/'>mutation-selection balance: <small>3</small></a></li>

	<li><a href='/tag/mutators/'>mutators: <small>2</small></a></li>

	<li><a href='/tag/pandoc/'>pandoc: <small>2</small></a></li>

	<li><a href='/tag/pdf/'>pdf: <small>1</small></a></li>

	<li><a href='/tag/phylogeny/'>phylogeny: <small>1</small></a></li>

	<li><a href='/tag/python/'>python: <small>2</small></a></li>

	<li><a href='/tag/recombination/'>recombination: <small>2</small></a></li>

	<li><a href='/tag/simulation/'>simulation: <small>2</small></a></li>

	<li><a href='/tag/st2/'>st2: <small>1</small></a></li>

	<li><a href='/tag/stress-induced%20mutation/'>stress-induced mutation: <small>1</small></a></li>

	<li><a href='/tag/twitter%20bootstrap/'>twitter bootstrap: <small>1</small></a></li>


							</ul>
						</li>			
						<li class="dropdown">
							<a href="#" class="dropdown-toggle" data-toggle="dropdown" title="Online avatars">
								<!-- Contact -->
								<i class="icon-user"></i>
								<b class="caret"></b>
							</a>
							<ul class="dropdown-menu">
								<!-- http://twitter.github.com/bootstrap/javascript.html#dropdowns -->
								
	
	<li><a href="http://www.yoavram.com/">Home <i class="icon-home"></i></a></li>
	
	
	<li><a href="mailto:yoavram@gmail.com">Email <i class="icon-envelope"></i></a></li>
	
	
	<li><a href="http://www.mendeley.com/profiles/yoav-ram">Mendeley</a></li>
	
	
	<li><a href="http://scholar.google.com/citations?user=RIFmJvYAAAAJ">Scholar Citations</a></li>
	
	
	<li><a href="http://figshare.com/authors/Yoav_Ram/99206">FigShare</a></li>
	
	
	<li><a href="http://twitter.com/yoavram">Twitter <i class="glyphicon-twitter-t"></i></a></li>
	
	
	<li><a href="http://www.linkedin.com/in/yoav-ram">Linkedin <i class="glyphicon-linked-in"></i></a></li>
	
	
	<li><a href="https://github.com/yoavram">GitHub <i class="glyphicon-github"></i></a></li>
	
	
	<li><a href="https://bitbucket.org/yoavram">Bitbucket</a></li>
	
	
	<li><a href="http://code.google.com/u/yoav-ram/">Google Code</a></li>
		
				
							</ul>
						</li>
						<li>
							<a href="http://www.yoavram.com/"  title="About me">
								<i class="icon-question-sign"></i>
								&nbsp;&nbsp;
							</a>
						</li>
						<li>
							<a href="http://feeds.feedburner.com/yoavram/msb-recent" rel="alternate" type="application/rss+xml" title="Subscribe in a reader">
								<i class="glyphicon-rss"></i>
								&nbsp;&nbsp;				
							</a>
						</li>
						<li>
							<a href="http://feedburner.google.com/fb/a/mailverify?uri=yoavram/msb-recent" title="Subscribe by Email">
								<i class="icon-envelope"></i>
								&nbsp;&nbsp;				
							</a>
						</li>		
					</ul>
				</div>
			</div>
		</div>
	</div>
	<!-- end Navbar -->

	<!-- HEADER -->
	
<div class="row-fluid">
	<div class="span7 offset2">
	<header class="masthead">
	
	<ul class="pager">
	  
	  <li class="previous">
		<a href='/mean-fitness-at-the-mutation-selection-balance/'>&larr; Older</a>
	  
	  </li>
	  
	  
	   
	   <li class="next">
		<a href='/convergence-of-mean-fitness/'>Newer &rarr;</a>
		
	  </li>
	</ul>

	
	<ul class="pager">
	<li class="previous">
		<a href='/'>&larr; Back to blog</a>
	</li>
	<li class="next">
		
		<a href="#bottom">To bottom &darr;</a>
		
	</li>
	</ul>
	<!-- end of pagination -->

</div>
</div>
<div class="row-fluid">
	<div class="span7 offset2">
	<h1>Summary: &#34;Mutators and sex in bacteria: Conflict between adaptive strategies&#34; (Tenaillon et al. 2000)</h1>
	
<ul class="breadcrumb" style="background-color: #fff;padding: 0px 0;margin: 0 0 10px;">
	<li class="active"><i class="icon-time"></i> 16 October 2012 at 13:22</li>
	
	<li class="active"><span class="divider">/</span>
	<i class="icon-refresh"></i> 14 January 2013 at 07:51</li>
	
	<li><span class="divider">/</span>
	<i class="icon-star"></i> <a href='http://blog.yoavram.com/tenaillon-et-al-2000/'>Permalink</a></li>
	
	
	<li><span class="divider">/</span>
		<i class="icon-tag"></i> <a href="/tag/asexual%20populations/">asexual populations</a></li>
	
	<li><span class="divider">/</span>
		<i class="icon-tag"></i> <a href="/tag/evolution%20of%20the%20mutation%20rate/">evolution of the mutation rate</a></li>
	
	<li><span class="divider">/</span>
		<i class="icon-tag"></i> <a href="/tag/mutators/">mutators</a></li>
	
	<li><span class="divider">/</span>
		<i class="icon-tag"></i> <a href="/tag/recombination/">recombination</a></li>
	
	<li><span class="divider">/</span>
		<i class="icon-tag"></i> <a href="/tag/simulation/">simulation</a></li>
	
	
	
	<li><span class="divider">/</span>
	
	<!-- sharethis buttons http://sharethis.com/publishers/get-sharing-tools -->
	<span class='st_email' ></span>
	<span class='st_twitter' ></span>
	<span class='st_facebook' ></span>
	<span class='st_googleplus' ></span>
	<span class='st_reddit' ></span>
	<span class='st_linkedin' ></span>
	<span class='st_stumbleupon' ></span>
	<span class='st_delicious' ></span>
	<span class='st_citeulike' ></span>
	<span class='st_connotea' ></span>

	</li>
	</ul>

	</header>
	</div>
	<div class="span3">
		
		
<div class="well-large">
	<strong>Please cite this article as:</strong>
	<br>
	Ram, Yoav (2012) Summary: &#34;Mutators and sex in bacteria: Conflict between adaptive strategies&#34; (Tenaillon et al. 2000). <em>The Mutation-Selection Blog</em>. 
	<a href='http://blog.yoavram.com/tenaillon-et-al-2000/'>http://blog.yoavram.com/tenaillon-et-al-2000/</a>
</div>

		
	</div>
</div>


	<!-- BODY -->	
	<div class="row-fluid">
		<div class="span7 offset2">
			

<div class="post">
	
    <ul>
<li><a href="#overview">Overview</a></li>
<li><a href="#model">Model</a><ul>
<li><a href="#genome">Genome</a></li>
<li><a href="#recombination">Recombination</a></li>
<li><a href="#parameters">Parameters</a></li>
</ul></li>
<li><a href="#stats">Stats</a></li>
<li><a href="#results">Results</a><ul>
<li><a href="#recombination-reduces-mutator-fixation">Recombination reduces mutator fixation</a></li>
<li><a href="#recombination-barriers">Recombination barriers</a></li>
<li><a href="#invasion-to-a-moderate-mutator-background">Invasion to a moderate mutator background</a></li>
<li><a href="#recombination-accelarates-adaptation">Recombination accelarates adaptation</a></li>
</ul></li>
<li><a href="#discussion">Discussion</a></li>
<li><a href="#references">References</a></li>
</ul>
<p><a name='TOC'></a></p>
<h2 id="overview"><a href="#overview">Overview</a></h2>
<p>This post is mostly a technical summary of the paper by <strong><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/">Tenaillon, Le Nagard, Godelle and Taddei <span class="citation">(2000)</span></a></strong>. I wrote the summary because I use it as a baseline for my own research, which involves the evolution of stress-induced mutators <span class="citation">(Ram and Hadany 2012)</span>.</p>
<p>The hypothesis the paper deals with is:</p>
<blockquote>
<p><em>The existence of genetic exchanges could modify the dynamics of adaptation and limits the success of mutator alleles<sup><a href="#fn1" class="footnoteRef" id="fnref1">1</a></sup></em>.</p>
</blockquote>
<p>It is stated that previous models <span class="citation">(Leigh 1973; Johnson 1999)</span> have considered mutators in a sexual population, but didn't consider the case of adaptation with <strong>multiple beneficial mutations</strong> - which can involve different <em>Hill-Robertosn effects</em> <span class="citation">(Hill and Robertson 1966)</span>.</p>
<p>This hypothesis was tested using simulations. The main conclusion, in comparison to a previous work in which the authors studies the evolution of mutators <span class="citation">(Tenaillon et al. 1999)</span>:</p>
<blockquote>
<p><em>Rare genetic exchanges have a dramatic (i.e. negative) effect on the fixation of mutators.</em></p>
</blockquote>
<p>In addition, the paper presents some evidence on <em>why</em> genetic exchanges (recombination) have such an effect of the evolution of mutators.</p>
<h2 id="model"><a href="#model">Model</a></h2>
<p>The model is an extension of the density-based model<sup><a href="#fn2" class="footnoteRef" id="fnref2">2</a></sup> used in <span class="citation">(Tenaillon et al. 1999)</span>. The extension is the addition of genetic exchanges (&quot;bacterial recombination&quot;). Both <strong>conjugation</strong> and <strong>transformation</strong> were used, but only the results of transformation are shown.</p>
<p>The model is built using:</p>
<ul>
<li>Finite populations - genetic drift, clonal interference</li>
<li>Directional selection - no epistasis, no changing environments</li>
<li>Several mutations with various fitness effects</li>
<li>Colonization of a new environment by a single cell - adaptive evolution scenario</li>
</ul>
<h3 id="genome"><a href="#genome">Genome</a></h3>
<p>The basic model of the genome (in parentheses are the values used in my own work for comparison):</p>
<ul>
<li>1 mutator locus (1)</li>
<li>6 loci for beneficial mutations (4+)
<ul>
<li>with additive effect on fitness: 0.06, 0.04, 0.04, 0.03, 0.03, 0.03 - mean 0.03833 (0.1)</li>
<li>all loci treated independently, only one locus will be involved in the process at one time (?)</li>
</ul></li>
<li>1,000 loci for deleterious mutations (+-1,000)
<ul>
<li>with additive effect on fitness: 0.05 (0.1)</li>
</ul></li>
</ul>
<h3 id="recombination"><a href="#recombination">Recombination</a></h3>
<ul>
<li>Drawing the number of recieved genes from a <em>Poisson</em> distribution</li>
<li>Randomly choose the sites, according to the number of sites drawn</li>
<li>Choosing a random allele for each site based on the allele frequency in the population</li>
<li>Constant gene transfer rate</li>
</ul>
<h3 id="parameters"><a href="#parameters">Parameters</a></h3>
<p>Rates are modified from the original to units of <em>events per genome per generation</em>. (in parentheses are the rates used in my own work for comparison):</p>
<ul>
<li>Mutators increase rates: 100-fold (2-10)</li>
<li>Beneficial mutations: 6x10<sup>-8</sup> based on 10<sup>-8</sup> per mutation (1.2x10<sup>-5</sup> - 3x10<sup>-5</sup> - x200)</li>
<li>Deleterious mutations: 10<sup>-4</sup> <span class="citation">(Kibota and Lynch 1996)</span> (3x10<sup>-3</sup> - 2.97x10<sup>-3</sup> <span class="citation">(Drake 1991)</span>- x30)</li>
<li>Lethal mutations: 10<sup>-5</sup> (0)</li>
<li>Non-mutator to mutator: 5x10<sup>-6</sup> (0)</li>
<li>Mutator to non-mutator: 100x5x10<sup>-10</sup> = 5x10<sup>-8</sup> (0)</li>
<li>Genetic exchanges: 0, 0.00001, 0.0001, 0.001, 0.01 and 0.1 (0, 0.015 or 0.75)
<ul>
<li>For comparison, <em>E. coli</em> do ~5x10<sup>-5</sup> and <em>H. pylori</em> ~3x10<sup>-3</sup> <span class="citation">(Milkman and Bridges 1990; Falush et al. 2001)</span>.</li>
</ul></li>
<li>Population size: mostly 10<sup>9</sup>, also checked 10<sup>5</sup>-10<sup>10</sup> (10<sup>5</sup>).</li>
</ul>
<h2 id="stats"><a href="#stats">Stats</a></h2>
<ul>
<li>Time of adaptation: no. generations for 95% of population to carry all beneficial mutations</li>
<li>Mutator frequency: mean frequency at end of adaptation process over 1,000 replicates</li>
<li>Mutator contribution: when mutator frequency &lt; 50% but at least one mutation that appeared in a mutator is present in &gt;50% of population</li>
<li>Mutator fixation: frequency &gt;50%</li>
</ul>
<h2 id="results"><a href="#results">Results</a></h2>
<h3 id="recombination-reduces-mutator-fixation"><a href="#recombination-reduces-mutator-fixation">Recombination reduces mutator fixation</a></h3>
<p>When the rate of genetic exchanges increases, the fixation probability of mutators decreases. <em>Figure 1A</em> shows this very clearly for a 100-fold mutator:</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/figure/F1/"><img src="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/bin/pq1800633001.jpg" alt="Figure 1" /></a></p>
<blockquote>
<p>Figure 1 from @Tenaillon2000a. A) Fixation probability of a 100-fold fold mutator, population size 10^-9. B) Adaptation times of populations with: circle - low mutation rate, square - high mutation rate, triangle - mixed populations.</p>
</blockquote>
<p>However, the <em>B panel</em> shows that the adaptation time of a mutator population is still shorter than that of a non-mutator population, even when mutators are loosing. This is due to the contribution of mutators to adaptation - in a recombining population mutators are <strong>&quot;altruistic&quot;</strong> alleles because they give away the beneficial mutations they generate. This causes them to lose their adaptive advantage and therefore their fixation probability drops, but it still increases the ability of the whole population to adapt.</p>
<p>Note that the rates of recombination in the figure are <em>per gene</em>. Converted to <em>per genome</em> they would be: 0, 10<sup>-5</sup>, 10<sup>-4</sup>, 10<sup>-3</sup>, 10<sup>-2</sup>, 10<sup>-1</sup>, or in decimal representation: 0, 0.00001, 0.0001, 0.001, 0.01, 0.1. Another useful presentation is the <em>ratio of recombination to mutation</em>, using the highest genomic mutation rate (10<sup>-4</sup>): 0, 0.1, 1, 10, 100, 1000, or compared to the 100-fold mutator: 0, 0.001, 0.01, 0.1, 1, 10.</p>
<p>So we see that as soon as the <em>recombination to mutation ratio</em> is 0.1 or lower (4<sup>th</sup> column from the left in <em>Fig. 1A</em>, if we consider the 100-fold increae), the fixation probability of mutators is ~50% or more. The conclusion here could be that mutators can evolve in a recombining population as long as recombination in not more than 10-fold stronger than mutation</p>
<p>Of course this <em>10-fold</em> estimate is highly dependent on the other paramters: the population size, the selection coefficients, the beneficial mutation rate, not to mention parameters that are not dealt with here such as epistasis.</p>
<h3 id="recombination-barriers"><a href="#recombination-barriers">Recombination barriers</a></h3>
<p>A recombination barrier, in the context of these simulations, prevents recombination between mutator and non-mutator individuals, limiting recombination events to within the sub-populations.</p>
<p>This is useful here because it allows to test if the source of the effect of recombination on the fixation of mutators is: <strong>1) breaking the linkage</strong> of the mutator with the beneficial mutations it generates, or <strong>2) competing adaptive strategy</strong>.</p>
<p>The 1<sup>st</sup> effect is usually called the <strong>hitch-hiking effect</strong>, or <strong>selective sweeps</strong> <span class="citation">(Charlesworth 2007)</span>: mutators can &quot;hitch-hike&quot; with the beneficial mutations they generate to high frequencies as long as the LD (linkage disequilibrium) between them is intact. recombination tends to break this LD, robbing the mutators from their precious beneficial mutations. Even worse, recombination doesn't only take away the beneifical mutations, it even gives them to the mutators' &quot;rivals&quot; - the competing non-mutator alleles.</p>
<p>The 2<sup>nd</sup> effect is general - <em>competing adadptive strategy</em>. How is recombination an <em>adaptive strategy</em>?</p>
<p>Asexual populations are limited by <strong>clonal interference</strong> <span class="citation">(Gerrish and Lenski 1998; Martens and Hallatschek 2011)</span>. The following figure illustrates the idea:</p>
<p><a href="http://en.wikipedia.org/wiki/Clonal_interference"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/5/5f/Evolsex-dia2a.svg/500px-Evolsex-dia2a.svg.png" alt="Clonal Interference" /></a></p>
<blockquote>
<p>Clonal interference diagram from <a href="http://en.wikipedia.org/wiki/File:Evolsex-dia2a.svg">Wikipedia</a>. This diagram illustrates how sex might create novel genotypes more rapidly. Two beneficial mutations A and B occur at random. The two mutations are recombined rapidly in a sexual population (top), but in an asexual population (bottom) the two mutations must independently arise because of clonal interference. The effect of recombination and sex on the adaptation rate via the reduction of <em>clonal interference</em> is also called the <em>Fisher-Muller effect</em>.</p>
</blockquote>
<p>If the reduction in fixation probability is due to the 1<sup>st</sup> effect - separation of the mutators from the beneficial mutations they generate - then <em>recombination barriers</em> are expected to revert the outcome, at least partially, to that of asexual populations.</p>
<p>However, the results showed that <em>recombination barriers</em> didn't recover the fixation probability of mutators to their levels without recombination. No figure presents the data, but the text provides some information: 7% fixation instead of 60% with beneficial mutation rate 10<sup>-8</sup> and 0.3% instead of 16% with a rate of 10<sup>-9</sup>. The conclusion is therefore that the 1<sup>st</sup> effect is not enough to explain the reduction in the mutators adaptive advantage in the presence of recombination.</p>
<h3 id="invasion-to-a-moderate-mutator-background"><a href="#invasion-to-a-moderate-mutator-background">Invasion to a moderate mutator background</a></h3>
<p>To check that the 2<sup>nd</sup> effect is the determining effect, a 100-fold mutator invaded another mutator (instead of a non-mutator). The results are in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/figure/F2/">Figure 2</a> - in general the fixation probability of a 100-fold mutator drops rapidly when competing with a 10-fold mutator and even a 5-fold mutator.</p>
<p>This illustrates that competing adaptive strategies can reduce the fixation of mutators, and therefore that is is plausible that this is why recombination reduces the fixation of mutators.</p>
<h3 id="recombination-accelarates-adaptation"><a href="#recombination-accelarates-adaptation">Recombination accelarates adaptation</a></h3>
<p>Setting the recombination rate at 10<sup>-4</sup> gene conversions per individual per generations, the effect of other parameters was investigated:</p>
<h4 id="population-size"><a href="#population-size">Population size</a></h4>
<p>In asexuals, increasing the population size increases the fixation rate of mutators <span class="citation">(Tenaillon et al. 1999)</span>.</p>
<p>In recombining populations, this is not the case. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/figure/F3/">Figure 3A</a> shows that for a population size &lt;10<sup>6</sup>, recombination has a very small effect on the fixation of a 100-fold mutator. However, the trends of the fixation probability are completely different for population size &gt;10<sup>-6</sup>. In asexual populations it increases (rapidly after 10<sup>7</sup>), but in recombining populations it decreases and then increases a bit again at 10<sup>9</sup>, but not much, never surprassing 0.2.</p>
<h4 id="beneficial-mutation-rate"><a href="#beneficial-mutation-rate">Beneficial mutation rate</a></h4>
<p>The lower the beneficial mutation rate, the lower the fixation probability of mutators. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/figure/F4/">Figure 4</a> shows that the fixation probability of mutators starts to increase at a beneficial mutation rate of 10<sup>-9</sup> for asexual populations, but only at 10<sup>-8</sup> for sexual populations.</p>
<h4 id="selection-coefficient-of-beneficial-mutations"><a href="#selection-coefficient-of-beneficial-mutations">Selection coefficient of beneficial mutations</a></h4>
<p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/figure/F5/">Figure 5</a> shows that the negative effect of recombination of the fixation of mutators decreases when the selection coefficient of beneficial mutations increases.</p>
<p>For a selection coefficient of 0.01 <span class="citation">(Kibota and Lynch 1996)</span> recombination reduced the fixation probability of mutators 17-fold.</p>
<p>A selection coefficient of 0.1, used in my own work [@Ram2012a], falls between the two right most points of <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/figure/F5/">Figure 5</a> - 2.5 and 3 - and the fixation probability of a 100-fold mutator allele drops ~10-fold due to recombination at a rate of 10<sup>-4</sup> per gene per generation in a population with size 10<sup>9</sup>.</p>
<h2 id="discussion"><a href="#discussion">Discussion</a></h2>
<p>This paper is <em>packed</em> with interesting results. The effect of recombination on mutator evolution has not recieved alot of treatment using simulation (but see <span class="citation">Levin and Cornejo (2009)</span> and <span class="citation">Sloan and Panjeti (2010)</span> on which I hope to write some other time). So this is really interesting to someone interesed in the evolution of the mutation rate, even 12 years later.</p>
<p>This work shows that indeed recombination has an effect on mutator evolution, and that this effect is highly sensitive - it is (I'm concentrating on when the effect is weak because that's when mutators are still going to evolve):</p>
<ul>
<li>Weak in small populations (&lt;10<sup>7</sup>-10<sup>8</sup>)</li>
<li>Weak for high beneficial mutation rates (&gt;10<sup>-8</sup>-10<sup>-7</sup>)</li>
<li>Weaker for strong/intermediate selection</li>
</ul>
<p>It seems that in general the effect of recombination on mutator evolution (that is, on the evolution of high mutation rates) is weak when the adaptation rate of the population is low. In turn, the adaptation rate is a function of the <strong>supply of beneficial mutations</strong> which is a function of population size and beneficial mutation rate, and of <strong>fixation time of beneficial mutations</strong> which is a function of population size and selection strength.</p>
<p>Selection of mutators is <strong>second-order selection</strong>, in the sense that it indirectly affecting mutators. Because their results imply that second-order selection of mutators is very sensitive to the system parameters, <em>Tenaillon et al.</em> propose that it may be unable to optimize the rates of mutation - in the sense that mutators will not be selected even when increasing the mutation rate will be beneficial to the population, in terms of fitness and adaptation rate. This can be resolved by <strong>transient hypermutation</strong>:</p>
<blockquote>
<p>Sex would therefore help bacteria approach an optimized mutation rate strategy composed of shifts towards high mutation rates in phases of adaptation under strong selective pressure and rapid recovery of low mutation rate once adaptation is achieved.</p>
</blockquote>
<p>This view was also proposed by Ninio <span class="citation">(1991)</span> and expanded in a review from the same lab with the attractive title <em>The Rise and Fall of Mutator Bacteria</em> <span class="citation">(Giraud et al. 2001)</span>.</p>
<p>I'm not sure if and how these results would change if a weaker mutator (10-fold mutation rate increase?) was used instead of the strong mutator (100-fold). For example, a paper from the same lab <span class="citation">(Taddei et al. 1997)</span> showed that a 10-fold mutator is more successful in invading finite populations (see <a href="http://www.nature.com/nature/journal/v387/n6634/fig_tab/387700a0_F3.html">Figure 3A</a>).</p>
<p>I learned alot from this paper (and it predecessors <span class="citation">(Taddei et al. 1997; Tenaillon et al. 1999)</span>) about how to think of evolutionary simulations. And of course the science itself - the diverse network of interactions and effects between selection, drift, mutation and recombination.</p>
<h2 id="references"><a href="#references">References</a></h2>
<p>Charlesworth, Brian. 2007. &#8220;A Hitch-hiking Guide To the Genome: a Commentary on &#8217;The Hitch-hiking Effect of a Favourable Gene&#8217; by John Maynard Smith and John Haigh.&#8221; <em>Genetical Research</em> 89 (5-6) (dec): 389&#8211;90. doi:10.1017/S0016672308009580. <a href="http://www.ncbi.nlm.nih.gov/pubmed/18976526" title="http://www.ncbi.nlm.nih.gov/pubmed/18976526">http://www.ncbi.nlm.nih.gov/pubmed/18976526</a>.</p>
<p>Drake, John W. 1991. &#8220;A Constant Rate of Spontaneous Mutation in DNA-based Microbes.&#8221; <em>Proceedings of the National Academy of Sciences of the United States of America</em> 88 (16) (aug): 7160&#8211;4. doi:10.1073/pnas.88.16.7160. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=52253" title="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=52253">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=52253</a>.</p>
<p>Falush, Daniel, Christian Kraft, Nancy S. Taylor, Pelayo Correa, James G. Fox, Mark Achtman, and Sebastian Suerbaum. 2001. &#8220;Recombination and Mutation During Long-term Gastric Colonization by Helicobacter Pylori: Estimates of Clock Rates, Recombination Size, and Minimal Age.&#8221; <em>Proceedings of the National Academy of Sciences of the United States of America</em> 98 (26) (dec): 15056&#8211;61. doi:10.1073/pnas.251396098. <a href="http://www.pnas.org/cgi/content/abstract/98/26/15056 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=64982" title="http://www.pnas.org/cgi/content/abstract/98/26/15056 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=64982">http://www.pnas.org/cgi/content/abstract/98/26/15056 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=64982</a>.</p>
<p>Gerrish, Philip J., and Richard E. Lenski. 1998. &#8220;The Fate of Competing Beneficial Mutations in an Asexual Population.&#8221; <em>Genetica</em> 102-103 (0): 127&#8211;144&#8211;144. doi:10.1023/A:1017067816551. <a href="http://www.springerlink.com/content/np187h5878166310/" title="http://www.springerlink.com/content/np187h5878166310/">http://www.springerlink.com/content/np187h5878166310/</a>.</p>
<p>Giraud, Antoine, Ivan Matic, Olivier Tenaillon, Antonio Clara, Miroslav Radman, Michel Fons, and Fran&#231;ois Taddei. 2001. &#8220;Costs and Benefits of High Mutation Rates: Adaptive Evolution of Bacteria in the Mouse Gut.&#8221; <em>Science</em> 291 (5513) (mar): 2606&#8211;8. doi:10.1126/science.1056421. <a href="http://www.ncbi.nlm.nih.gov/pubmed/11283373" title="http://www.ncbi.nlm.nih.gov/pubmed/11283373">http://www.ncbi.nlm.nih.gov/pubmed/11283373</a>.</p>
<p>Hill, W. G., and Alan Robertson. 1966. &#8220;The Effect of Linkage on Limits To Artificial Selection.&#8221; <em>Genetical Research</em> 8 (3) (dec): 269&#8211;94. doi:10.1017/S0016672300010156. <a href="http://www.ncbi.nlm.nih.gov/pubmed/5980116 http://www.journals.cambridge.org/abstract\_S0016672300010156" title="http://www.ncbi.nlm.nih.gov/pubmed/5980116 http://www.journals.cambridge.org/abstract\_S0016672300010156">http://www.ncbi.nlm.nih.gov/pubmed/5980116 http://www.journals.cambridge.org/abstract\_S0016672300010156</a>.</p>
<p>Johnson, Toby. 1999. &#8220;Beneficial Mutations, Hitchhiking and the Evolution of Mutation Rates in Sexual Populations.&#8221; <em>Genetics</em> 151 (4) (apr): 1621&#8211;31. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1460574\&amp;tool=pmcentrez\&amp;rendertype=abstract" title="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1460574\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1460574\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</p>
<p>Kibota, Travis T., and Michael Lynch. 1996. &#8220;Estimate of the Genomic Mutation Rate Deleterious To Overall Fitness in E. Coli.&#8221; <em>Nature</em> 381 (6584) (jun): 694&#8211;6. doi:10.1038/381694a0. <a href="http://www.ncbi.nlm.nih.gov/pubmed/8649513" title="http://www.ncbi.nlm.nih.gov/pubmed/8649513">http://www.ncbi.nlm.nih.gov/pubmed/8649513</a>.</p>
<p>Leigh, Egbert Giles. 1973. &#8220;The Evolution of Mutation Rates.&#8221; <em>Genetics</em> 73 (apr): Suppl73:1&#8211;18. <a href="http://www.ncbi.nlm.nih.gov/pubmed/4711556" title="http://www.ncbi.nlm.nih.gov/pubmed/4711556">http://www.ncbi.nlm.nih.gov/pubmed/4711556</a>.</p>
<p>Levin, Bruce R., and Omar E. Cornejo. 2009. &#8220;The Population and Evolutionary Dynamics of Homologous Gene Recombination in Bacterial Populations.&#8221; <em>PLoS Genetics</em> 5 (8) (aug): e1000601. doi:10.1371/journal.pgen.1000601. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2717328\&amp;tool=pmcentrez\&amp;rendertype=abstract" title="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2717328\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2717328\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</p>
<p>Martens, Erik A., and Oskar Hallatschek. 2011. &#8220;Interfering Waves of Adaptation Promote Spatial Mixing.&#8221; <em>Genetics</em> 189 (3) (sep): 1045&#8211;1060. doi:10.1534/genetics.111.130112. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21900264" title="http://www.ncbi.nlm.nih.gov/pubmed/21900264">http://www.ncbi.nlm.nih.gov/pubmed/21900264</a>.</p>
<p>Milkman, Roger, and Mellissa McKane Bridges. 1990. &#8220;Molecular Evolution of the Escherichia Coli Chromosome. III. Clonal Frames.&#8221; <em>Genetics</em> 126 (3) (nov): 505&#8211;17. <a href="http://mutage.oxfordjournals.org/cgi/doi/10.1093/mutage/2.2.147 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1204208" title="http://mutage.oxfordjournals.org/cgi/doi/10.1093/mutage/2.2.147 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1204208">http://mutage.oxfordjournals.org/cgi/doi/10.1093/mutage/2.2.147 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1204208</a>.</p>
<p>Ninio, Jacques. 1991. &#8220;Transient Mutators: a Semiquantitative Analysis of the Influence of Translation and Transcription Errors on Mutation Rates.&#8221; <em>Genetics</em> 129 (3) (nov): 957&#8211;62. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1204761\&amp;tool=pmcentrez\&amp;rendertype=abstract" title="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1204761\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1204761\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</p>
<p>Ram, Yoav, and Lilach Hadany. 2012. &#8220;The Evolution of Stress-induced Hypermutation in Asexual Populations.&#8221; <em>Evolution</em> 66 (7) (jul): 2315&#8211;28. doi:10.1111/j.1558-5646.2012.01576.x. <a href="http://doi.wiley.com/10.1111/j.1558-5646.2012.01576.x http://www.ncbi.nlm.nih.gov/pubmed/22759304" title="http://doi.wiley.com/10.1111/j.1558-5646.2012.01576.x http://www.ncbi.nlm.nih.gov/pubmed/22759304">http://doi.wiley.com/10.1111/j.1558-5646.2012.01576.x http://www.ncbi.nlm.nih.gov/pubmed/22759304</a>.</p>
<p>Sloan, Daniel B., and Vijay G. Panjeti. 2010. &#8220;Evolutionary Feedbacks Between Reproductive Mode and Mutation Rate Exacerbate the Paradox of Sex.&#8221; <em>Evolution</em> 64 (4) (apr): 1129&#8211;35. doi:10.1111/j.1558-5646.2009.00869.x. <a href="http://www.ncbi.nlm.nih.gov/pubmed/19863589" title="http://www.ncbi.nlm.nih.gov/pubmed/19863589">http://www.ncbi.nlm.nih.gov/pubmed/19863589</a>.</p>
<p>Taddei, Fran&#231;ois, Miroslav Radman, John Maynard Smith, Bruno Toupance, Pierre-Henri Gouyon, and Bernard Godelle. 1997. &#8220;Role of Mutator Alleles in Adaptive Evolution.&#8221; <em>Nature</em> 387 (6634) (jun): 700&#8211;2. doi:10.1038/42696. <a href="http://www.ncbi.nlm.nih.gov/pubmed/9192893" title="http://www.ncbi.nlm.nih.gov/pubmed/9192893">http://www.ncbi.nlm.nih.gov/pubmed/9192893</a>.</p>
<p>Tenaillon, Olivier, Herve Le Nagard, Bernard Godelle, and Fran&#231;ois Taddei. 2000. &#8220;Mutators and Sex in Bacteria: Conflict Between Adaptive Strategies.&#8221; <em>Proceedings of the National Academy of Sciences of the United States of America</em> 97 (19) (sep): 10465&#8211;70. doi:10.1073/pnas.180063397. <a href="http://www.ncbi.nlm.nih.gov/pubmed/10973474" title="http://www.ncbi.nlm.nih.gov/pubmed/10973474">http://www.ncbi.nlm.nih.gov/pubmed/10973474</a>.</p>
<p>Tenaillon, Olivier, Bruno Toupance, Herve Le Nagard, Fran&#231;ois Taddei, and Bernard Godelle. 1999. &#8220;Mutators, Population Size, Adaptive Landscape and the Adaptation of Asexual Populations of Bacteria.&#8221; <em>Genetics</em> 152 (2): 485&#8211;493. <a href="http://www.genetics.org/cgi/content/abstract/152/2/485" title="http://www.genetics.org/cgi/content/abstract/152/2/485">http://www.genetics.org/cgi/content/abstract/152/2/485</a>.</p>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>Alleles that increase the global mutation rate. For example, a mutant allele of the <em>mutS</em> gene can reduce the efficiency of DNA repair, allowing more mutations to slip though the repair mechanisms.<a href="#fnref1">&#8617;</a></p></li>
<li id="fn2"><p>A density-based model is one where the <em>number</em> of individuals in each class is monitored and modified from one generation to the next. This is in contrast to frequency-dependent models in which the frequency of individuals in each class is monitored.<a href="#fnref2">&#8617;</a></p></li>
</ol>
</div>
	<a href="#" id="download_bibtex_btn">Download BibTeX</a>
	<script type="text/javascript">
	$(function() {
		$('#download_bibtex_btn').click(function() {
			var content = $('#bib').html();
			var w = window.open();
			$(w.document.body).text(content);
			w.document.title = 'Summary: &#34;Mutators and sex in bacteria: Conflict between adaptive strategies&#34; (Tenaillon et al. 2000)' + '.bib';
			w.document.contentType = 'text/x-bibtex';
		});
	})
	</script>
	<a name="bottom"></a>
	
	
	<ul class="pager">
	  
	  <li class="previous">
		<a href='/mean-fitness-at-the-mutation-selection-balance/'>&larr; Older</a>
	  
	  </li>
	  
	  
	   
	   <li class="next">
		<a href='/convergence-of-mean-fitness/'>Newer &rarr;</a>
		
	  </li>
	</ul>

	
	<ul class="pager">
	<li class="previous">
		<a href='/'>&larr; Back to blog</a>
	</li>
	<li class="next">
		
		<a href="#">To top &uarr;</a>
		
	</li>
	</ul>
	<!-- end of pagination -->

	
</div>
<div id="bib" style="display: none">
	
@article{Levin2009,
abstract = {In bacteria, recombination is a rare event, not a part of the reproductive process. Nevertheless, recombination -- broadly defined to include the acquisition of genes from external sources, i.e., horizontal gene transfer (HGT) -- plays a central role as a source of variation for adaptive evolution in many species of bacteria. Much of niche expansion, resistance to antibiotics and other environmental stresses, virulence, and other characteristics that make bacteria interesting and problematic, is achieved through the expression of genes and genetic elements obtained from other populations of bacteria of the same and different species, as well as from eukaryotes and archaea. While recombination of homologous genes among members of the same species has played a central role in the development of the genetics and molecular biology of bacteria, the contribution of homologous gene recombination (HGR) to bacterial evolution is not at all clear. Also, not so clear are the selective pressures responsible for the evolution and maintenance of transformation, the only bacteria-encoded form of HGR. Using a semi-stochastic simulation of mutation, recombination, and selection within bacterial populations and competition between populations, we explore (1) the contribution of HGR to the rate of adaptive evolution in these populations and (2) the conditions under which HGR will provide a bacterial population a selective advantage over non-recombining or more slowly recombining populations. The results of our simulation indicate that, under broad conditions: (1) HGR occurring at rates in the range anticipated for bacteria like Streptococcus pneumoniae, Escherichia coli, Haemophilus influenzae, and Bacillus subtilis will accelerate the rate at which a population adapts to environmental conditions; (2) once established in a population, selection for this capacity to increase rates of adaptive evolution can maintain bacteria-encoded mechanisms of recombination and prevent invasion of non-recombining populations, even when recombination engenders a modest fitness cost; and (3) because of the density- and frequency-dependent nature of HGR in bacteria, this capacity to increase rates of adaptive evolution is not sufficient as a selective force to provide a recombining population a selective advantage when it is rare. Under realistic conditions, homologous gene recombination will increase the rate of adaptive evolution in bacterial populations and, once established, selection for higher rates of evolution will promote the maintenance of bacteria-encoded mechanisms for HGR. On the other hand, increasing rates of adaptive evolution by HGR is unlikely to be the sole or even a dominant selective pressure responsible for the original evolution of transformation.},
annote = {-semi-stochastic simulations: finite size and monte carlo decisions, but not individual-based (so very small genome).
-We demonstrate that under broad conditions, HGR occurring at rates in a range estimated for E. coli, H. influenza, S. pneumoniae,, and B. subtilis can increase the rate of adaptive evolution in bacterial populations. We
-the conditions under which recombination can provide a population a selective advantage in competition with non-recombining populations are restricted to when the recombining population is relatively common and the total population density is high
-when the recombining population is rare, it will not be favored despite its ability to acquire genes from the dominant non- recombining population.
-without cost, recombining pops win, with 1\% cost they lose, but only when there is no initial variation. with variation they win despite costs and despite low freq at beginning: this is because they doo not wait for mutaions. },
author = {Levin, Bruce R. and Cornejo, Omar E.},
doi = {10.1371/journal.pgen.1000601},
file = {:D$\backslash$:/papers/Levin, Cornejo - 2009 - The population and evolutionary dynamics of homologous gene recombination in bacterial populations.pdf:pdf},
issn = {1553-7404},
journal = {PLoS genetics},
keywords = {Bacteria,Bacteria: genetics,Evolution,Genetic,Models,Molecular,Mutation,Recombination,Selection,bacteria,recombination},
mendeley-tags = {bacteria,recombination},
month = aug,
number = {8},
pages = {e1000601},
pmid = {19680442},
title = {{The population and evolutionary dynamics of homologous gene recombination in bacterial populations.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2717328\&tool=pmcentrez\&rendertype=abstract},
volume = {5},
year = {2009}
}
@article{Ram2012,
abstract = {Numerous empirical studies show that stress of various kinds induces a state of hypermutation in bacteria via multiple mechanisms, but theoretical treatment of this intriguing phenomenon is lacking. We used deterministic and stochastic models to study the evolution of stress-induced hypermutation in infinite and finite-size populations of bacteria undergoing selection, mutation, and random genetic drift in constant environments and in changing ones. Our results suggest that if beneficial mutations occur, even rarely, then stress-induced hypermutation is advantageous for bacteria at both the individual and the population levels and that it is likely to evolve in populations of bacteria in a wide range of conditions because it is favored by selection. These results imply that mutations are not, as the current view holds, uniformly distributed in populations, but rather that mutations are more common in stressed individuals and populations. Because mutation is the raw material of evolution, these results have a profound impact on broad aspects of evolution and biology.},
author = {Ram, Yoav and Hadany, Lilach},
doi = {10.1111/j.1558-5646.2012.01576.x},
file = {:D$\backslash$:/papers/Ram, Hadany - 2012 - The evolution of stress-induced hypermutation in asexual populations.pdf:pdf;:D$\backslash$:/papers//Ram, Hadany - 2012 - The evolution of stress-induced hypermutation in asexual populations.pdf:pdf},
issn = {1558-5646},
journal = {Evolution},
keywords = {Adaptation,Asexual,Bacteria,Bacteria: genetics,Bacteria: growth \& development,Biological,Computer Simulation,Environment,Genetic,Genetic Drift,Models,Mutation Rate,Physiological,Population Density,Reproduction,Stochastic Processes,Stress},
month = jul,
number = {7},
pages = {2315--28},
pmid = {22759304},
title = {{The evolution of stress-induced hypermutation in asexual populations.}},
url = {http://doi.wiley.com/10.1111/j.1558-5646.2012.01576.x http://www.ncbi.nlm.nih.gov/pubmed/22759304},
volume = {66},
year = {2012}
}
@article{Tenaillon1999,
abstract = {Selection of mutator alleles, increasing the mutation rate up to 10,000-fold, has been observed during in vitro experimental evolution. This spread is ascribed to the hitchhiking of mutator alleles with favorable mutations, as demonstrated by a theoretical model using selective parameters corresponding to such experiments. Observations of unexpectedly high frequencies of mutators in natural isolates suggest that the same phemonemon could occur in the wild. But it remains questionable whether realistic in natura parameter values could also result in selection of mutators. In particular, the main parameters of adaptation, the size of the adapting population and the height and steepness of the adaptive peak characterizing adaptation, are very variable in nature. By simulation approach, we studied the effect of these parameters on the selection of mutators in asexual populations, assuming additive fitness. We show that the larger the population size, the more likely the fixation of mutator alleles. At a large population size, at least four adaptive mutations are needed for mutator fixation; moreover, under stronger selection stronger mutators are selected. We propose a model based on multiple mutations to illustrate how second-order selection can optimize population fitness when few favorable mutations are required for adaptation.},
annote = {treis to evaluate the results of Taddei et al 1997 for "natural" parameter values:
*population size
*number of beneficial mutations needed for adaptation
*selection coefficient of beneficial mutations
        
        
probability of fixation is defined as the percantage of populations in which the frequency of the mutator  at the end of adaptation reaches 95\% - much harder for the mutator then in our model},
author = {Tenaillon, Olivier and Toupance, Bruno and {Le Nagard}, Herve and Taddei, Fran\c{c}ois and Godelle, Bernard},
file = {:D$\backslash$:/papers/Tenaillon et al. - 1999 - Mutators, Population Size, Adaptive Landscape and the Adaptation of Asexual Populations of Bacteria.pdf:pdf},
journal = {Genetics},
keywords = {asexual,bacteria,mutation rate,simulation},
mendeley-tags = {asexual,bacteria,mutation rate,simulation},
number = {2},
pages = {485--493},
title = {{Mutators, Population Size, Adaptive Landscape and the Adaptation of Asexual Populations of Bacteria}},
url = {http://www.genetics.org/cgi/content/abstract/152/2/485},
volume = {152},
year = {1999}
}
@article{Ninio1991,
abstract = {A population of bacteria growing in a nonlimiting medium includes mutator bacteria and transient mutators defined as wild-type bacteria which, due to occasional transcription or translation errors, display a mutator phenotype. A semiquantitative theoretical analysis of the steady-state composition of an Escherichia coli population suggests that true strong genotypic mutators produce about 3 x 10(-3) of the single mutations arising in the population, while transient mutators produce at least 10\% of the single mutations and more than 95\% of the simultaneous double mutations. Numbers of mismatch repair proteins inherited by the offspring, proportions of lethal mutations and mortality rates are among the main parameters that influence the steady-state composition of the population. These results have implications for the experimental manipulation of mutation rates and the evolutionary fixation of frequent but nearly neutral mutations (e.g., synonymous codon substitutions).},
annote = {transient mutator-phenotype of mutator with wild-type genotype responsible for most mutations and accelerates rate of double mutants.
 semi-quantitative approached
 parameter valves},
author = {Ninio, Jacques},
file = {:D$\backslash$:/papers/Ninio - 1991 - Transient mutators a semiquantitative analysis of the influence of translation and transcription errors on mutation rates.pdf:pdf},
issn = {0016-6731},
journal = {Genetics},
keywords = {Bacteria,Bacteria: genetics,DNA Repair,DNA Replication,Genetic,Genotype,Mutagenesis,Phenotype,Protein Biosynthesis,Transcription},
month = nov,
number = {3},
pages = {957--62},
pmid = {1752431},
title = {{Transient mutators: a semiquantitative analysis of the influence of translation and transcription errors on mutation rates.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1204761\&tool=pmcentrez\&rendertype=abstract},
volume = {129},
year = {1991}
}
@article{Martens2011,
abstract = {A fundamental problem of asexual adaptation is that beneficial substitutions are not efficiently accumulated in large populations: Beneficial mutations often go extinct because they compete with one another in going to fixation. It has been argued that such clonal interference may have led to the evolution of sex and recombination in well-mixed populations. Here, we study clonal interference, and mechanisms of its mitigation, in an evolutionary model of spatially structured populations with uniform selection pressure. Clonal interference is much more prevalent with spatial structure than without, due to the slow wave-like spread of beneficial mutations through space. We find that the adaptation speed of asexuals saturates when the linear habitat size exceeds a characteristic interference length, which becomes shorter with smaller migration and larger mutation rate. The limiting speed is proportional to $\mu$(1/2) and $\mu$(1/3) in linear and planar habitats, respectively, where the mutational supply $\mu$ is the product of mutation rate and local population density. This scaling and the existence of a speed limit should be amenable to experimental tests as they fall far below predicted adaptation speeds for well-mixed populations (that scale as the logarithm of population size). Finally, we show that not only recombination, but also long-range migration is a highly efficient mechanism of relaxing clonal competition in structured populations. Our conservative estimates of the interference length predict prevalent clonal interference in microbial colonies and biofilms, so clonal competition should be a strong driver of both genetic and spatial mixing in those contexts.},
annote = {What is the effect of interfering Fisher waves on the speed of adaptation and the genetic diversity in an asexual popula- tion? Are there simple mechanisms of mitigating clonal interference and thus accelerating adaptation? Are these effects relevant to microbial colonies and biofilms, and perhaps measurable in evolution experiments?},
author = {Martens, Erik A. and Hallatschek, Oskar},
doi = {10.1534/genetics.111.130112},
file = {:D$\backslash$:/papers/Martens, Hallatschek - 2011 - Interfering Waves of Adaptation Promote Spatial Mixing.pdf:pdf},
issn = {1943-2631},
journal = {Genetics},
keywords = {Fisher-Muller effect,asexual,beneficial mutations,clonal interference,structured populations},
mendeley-tags = {Fisher-Muller effect,asexual,beneficial mutations,clonal interference,structured populations},
month = sep,
number = {3},
pages = {1045--1060},
pmid = {21900264},
title = {{Interfering Waves of Adaptation Promote Spatial Mixing.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/21900264},
volume = {189},
year = {2011}
}
@article{Charlesworth2007,
author = {Charlesworth, Brian},
doi = {10.1017/S0016672308009580},
file = {:D$\backslash$:/papers/Charlesworth - 2007 - A hitch-hiking guide to the genome a commentary on 'The hitch-hiking effect of a favourable gene' by John Maynard Smith and John Haigh.pdf:pdf},
isbn = {0016672308009},
journal = {Genetical research},
keywords = {Animals,Genetic,Genetic Variation,Genome,Humans,Isoenzymes,Isoenzymes: genetics,Selection,hitch-hiking,review},
mendeley-tags = {hitch-hiking,review},
month = dec,
number = {5-6},
pages = {389--90},
pmid = {18976526},
title = {{A hitch-hiking guide to the genome: a commentary on 'The hitch-hiking effect of a favourable gene' by John Maynard Smith and John Haigh.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/18976526},
volume = {89},
year = {2007}
}
@article{Johnson1999,
abstract = {Natural selection acts in three ways on heritable variation for mutation rates. A modifier allele that increases the mutation rate is (i) disfavored due to association with deleterious mutations, but is also favored due to (ii) association with beneficial mutations and (iii) the reduced costs of lower fidelity replication. When a unique beneficial mutation arises and sweeps to fixation, genetic hitchhiking may cause a substantial change in the frequency of a modifier of mutation rate. In previous studies of the evolution of mutation rates in sexual populations, this effect has been underestimated. This article models the long-term effect of a series of such hitchhiking events and determines the resulting strength of indirect selection on the modifier. This is compared to the indirect selection due to deleterious mutations, when both types of mutations are randomly scattered over a given genetic map. Relative to an asexual population, increased levels of recombination reduce the effects of beneficial mutations more rapidly than those of deleterious mutations. However, the role of beneficial mutations in determining the evolutionarily stable mutation rate may still be significant if the function describing the cost of high-fidelity replication has a shallow gradient.},
author = {Johnson, Toby},
file = {:D$\backslash$:/papers/Johnson - 1999 - Beneficial mutations, hitchhiking and the evolution of mutation rates in sexual populations.pdf:pdf},
issn = {0016-6731},
journal = {Genetics},
keywords = {Alleles,Animals,Biological Evolution,Female,Genetic,Male,Models,Mutation,Reproduction,Reproduction: genetics,Selection,mutation rate,recombination,theory},
mendeley-tags = {mutation rate,recombination,theory},
month = apr,
number = {4},
pages = {1621--31},
pmid = {10101182},
title = {{Beneficial mutations, hitchhiking and the evolution of mutation rates in sexual populations.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1460574\&tool=pmcentrez\&rendertype=abstract},
volume = {151},
year = {1999}
}
@article{Giraud2001,
abstract = {We have shown that bacterial mutation rates change during the experimental colonization of the mouse gut. A high mutation rate was initially beneficial because it allowed faster adaptation, but this benefit disappeared once adaptation was achieved. Mutator bacteria accumulated mutations that, although neutral in the mouse gut, are often deleterious in secondary environments. Consistently, the competitiveness of mutator bacteria is reduced during transmission to and re-colonization of similar hosts. The short-term advantages and long-term disadvantages of mutator bacteria could account for their frequency in nature.},
annote = {"The mouse model showed that the advan-tage of mutator bacteria when colonizing new host is due to their capacity to generate adap- tive mutations rapidly, allowing them to ex- ploit the ecosystem resources more quickly than wild-type bacteria. This advantage is reduced to little or nothing once adaptation is achieved. Moreover, if the mutation rate is not reduced [as observed in some subpopula- tions (Fig. 1A)], it leads progressively to loss of functions that are dispensable in the cur- rent environment but compromise the long- term survival of mutator clones."},
author = {Giraud, Antoine and Matic, Ivan and Tenaillon, Olivier and Clara, Antonio and Radman, Miroslav and Fons, Michel and Taddei, Fran\c{c}ois},
doi = {10.1126/science.1056421},
file = {:D$\backslash$:/papers/Giraud et al. - 2001 - Costs and benefits of high mutation rates adaptive evolution of bacteria in the mouse gut.pdf:pdf},
issn = {0036-8075},
journal = {Science},
keywords = {Adaptation,Adenosine Triphosphatases,Animals,Bacterial,Bacterial Proteins,Bacterial Proteins: genetics,Base Pair Mismatch,Biological Evolution,DNA Repair,DNA Repair: genetics,DNA-Binding Proteins,Escherichia coli,Escherichia coli Proteins,Escherichia coli: genetics,Escherichia coli: growth \& development,Escherichia coli: physiology,Feces,Feces: microbiology,Genes,Germ-Free Life,Inbred C3H,Intestines,Intestines: microbiology,Mice,MutS DNA Mismatch-Binding Protein,Mutation,Physiological,in vivo,mutation rate},
mendeley-tags = {in vivo,mutation rate},
month = mar,
number = {5513},
pages = {2606--8},
pmid = {11283373},
title = {{Costs and benefits of high mutation rates: adaptive evolution of bacteria in the mouse gut.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/11283373},
volume = {291},
year = {2001}
}
@article{Gerrish1998,
abstract = {In sexual populations, beneficial mutations that occur in different lineages may be recombined into a single lineage. In asexual populations, however, clones that carry such alternative beneficial mutations compete with one another and, thereby, interfere with the expected progression of a given mutation to fixation. From theoretical exploration of such clonal interference, we have derived (1) a fixation probability for beneficial mutations, (2) an expected substitution rate, (3) an expected coefficient of selection for realized substitutions, (4) an expected rate of fitness increase, (5) the probability that a beneficial mutation transiently achieves polymorphic frequency ( 1\%), and (6) the probability that a beneficial mutation transiently achieves majority status. Based on (2) and (3), we were able to estimate the beneficial mutation rate and the distribution of mutational effects from changes in mean fitness in an evolving E. coli population.},
annote = {derivations for the probability that a beneficial mutation reaches some frequency (incl. fixation) in an asexual population
including paramter estimation
can be used to check simulation results},
author = {Gerrish, Philip J. and Lenski, Richard E.},
doi = {10.1023/A:1017067816551},
file = {:D$\backslash$:/papers/Gerrish, Lenski - 1998 - The fate of competing beneficial mutations in an asexual population.pdf:pdf},
issn = {0016-6707},
journal = {Genetica},
keywords = {asexual,asexual population dynamics,beneficial mutations,clonal interference,fixation probability,substitution rate},
mendeley-tags = {asexual,beneficial mutations,clonal interference},
number = {0},
pages = {127--144--144},
publisher = {Springer Netherlands},
title = {{The fate of competing beneficial mutations in an asexual population}},
url = {http://www.springerlink.com/content/np187h5878166310/},
volume = {102-103},
year = {1998}
}
@article{Falush2001,
abstract = {The bacterium Helicobacter pylori colonizes the gastric mucosa of half of the human population, resulting in chronic gastritis, ulcers, and cancer. We sequenced ten gene fragments from pairs of strains isolated sequentially at a mean interval of 1.8 years from 26 individuals. Several isolates had acquired small mosaic segments from other H. pylori or point mutations. The maximal mutation rate, the import size, and the frequency of recombination were calculated by using a Bayesian model. The calculations indicate that the last common ancestor of H. pylori existed at least 2,500-11,000 years ago. Imported mosaics have a median size of 417 bp, much smaller than for other bacteria, and recombination occurs frequently (60 imports spanning 25,000 bp per genome per year). Thus, the panmictic population structure of H. pylori results from very frequent recombination during mixed colonization by unrelated strains.},
author = {Falush, Daniel and Kraft, Christian and Taylor, Nancy S. and Correa, Pelayo and Fox, James G. and Achtman, Mark and Suerbaum, Sebastian},
doi = {10.1073/pnas.251396098},
file = {:D$\backslash$:/papers/Falush et al. - 2001 - Recombination and mutation during long-term gastric colonization by Helicobacter pylori estimates of clock rates, recombination size, and minimal age.pdf:pdf},
issn = {0027-8424},
journal = {Proceedings of the National Academy of Sciences of the United States of America},
keywords = {Bayes Theorem,Biological,Gastric Mucosa,Gastric Mucosa: microbiology,Genetic,Helicobacter pylori,Helicobacter pylori: genetics,Helicobacter pylori: growth \& development,Helicobacter pylori: isolation \& purification,Humans,Models,Molecular Sequence Data,Mutation,Recombination,bacteria,mutation rate,recombination rate},
mendeley-tags = {Helicobacter pylori,bacteria,mutation rate,recombination rate},
month = dec,
number = {26},
pages = {15056--61},
pmid = {11742075},
title = {{Recombination and mutation during long-term gastric colonization by Helicobacter pylori: estimates of clock rates, recombination size, and minimal age.}},
url = {http://www.pnas.org/cgi/content/abstract/98/26/15056 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=64982},
volume = {98},
year = {2001}
}
@article{Leigh1973,
author = {Leigh, Egbert Giles Jr.},
file = {:D$\backslash$:/papers/Leigh - 1973 - The evolution of mutation rates.pdf:pdf},
issn = {0016-6731},
journal = {Genetics},
keywords = {Adaptation,Alleles,Biological,Biological Evolution,Environment,Genes,Genetic,Lethal,Mutation,Selection,mutation rate,theory},
mendeley-tags = {mutation rate,theory},
month = apr,
pages = {Suppl 73:1--18},
pmid = {4711556},
title = {{The evolution of mutation rates.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/4711556},
volume = {73},
year = {1973}
}
@article{Taddei1997,
abstract = {Because most newly arising mutations are neutral or deleterious, it has been argued that the mutation rate has evolved to be as low as possible, limited only by the cost of error-avoidance and error-correction mechanisms. But up to one per cent of natural bacterial isolates are 'mutator' clones that have high mutation rates. We consider here whether high mutation rates might play an important role in adaptive evolution. Models of large, asexual, clonal populations adapting to a new environment show that strong mutator genes (such as those that increase mutation rates by 1,000-fold) can accelerate adaptation, even if the mutator gene remains at a very low frequency (for example, 10[-5]). Less potent mutators (10 to 100-fold increase) can become fixed in a fraction of finite populations. The parameters of the model have been set to values typical for Escherichia coli cultures, which behave in a manner similar to the model in long-term adaptation experiments.},
annote = {-transient mutators:  by mutation and reverse mutation at modifier gene. rate: \~{} 10-5-10-7
-if transient effect is removed, that is, no reversion allowed,  than mutators do not speed adaptation
-"Furthermore, these results show that during the course of evolution, where phases of adaptation and stasis alternate, there is no pure strategy of mutation rate. Rather, selection seems to result in an alternation of high and low mutation rate through forward and reverse mutations at the mutator locus. Alternatively, transient mutators
ability by increasing their mutation rate: for example, inducible mechanisms (see ref. 30 for a review) might be particularly useful in response to stress."
      },
author = {Taddei, Fran\c{c}ois and Radman, Miroslav and {Maynard Smith}, John and Toupance, Bruno and Gouyon, Pierre-Henri and Godelle, Bernard},
doi = {10.1038/42696},
file = {:D$\backslash$:/papers/Taddei et al. - 1997 - Role of mutator alleles in adaptive evolution.pdf:pdf},
issn = {0028-0836},
journal = {Nature},
keywords = {Adaptation,Alleles,Animals,Biological,Directed Molecular Evolution,Escherichia coli,Evolution,Genetic,Mammals,Models,Molecular,Mutation,Physiological,Selection,mutator},
mendeley-tags = {mutator},
month = jun,
number = {6634},
pages = {700--2},
pmid = {9192893},
title = {{Role of mutator alleles in adaptive evolution.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/9192893},
volume = {387},
year = {1997}
}
@article{Milkman1990,
abstract = {PCR fragments, 1500-bp, from 15 previously sequenced regions in the Escherichia coli chromosome have been compared by restriction analysis in a large set of wild (ECOR) strains. Prior published observations of segmental clonality are confirmed: each of several sequence types is shared by a number of strains. The rate of recombinational replacement and the average size of the replacements are estimated in a set of closely related strains in which a clonal frame is dotted with occasional stretches of DNA belonging to other clones. A clonal hierarchy is described. Some new comparative sequencing data are presented.},
annote = {r/m = 0.02
r=5x10-12 (recombination per nuc. per gen.)
        
mu = 3x10-10 (per nuc per gen)},
author = {Milkman, Roger and Bridges, Mellissa McKane},
file = {:D$\backslash$:/papers/Milkman, Bridges - 1990 - Molecular evolution of the Escherichia coli chromosome. III. Clonal frames.pdf:pdf},
issn = {0016-6731},
journal = {Genetics},
keywords = {Bacterial,Base Sequence,Biological Evolution,Chromosomes,Cloning,Escherichia coli,Escherichia coli: genetics,Molecular,Nucleic Acid,Operon,Polymerase Chain Reaction,Polymorphism,Restriction Fragment Length,Sequence Homology,recombination rate},
mendeley-tags = {recombination rate},
month = nov,
number = {3},
pages = {505--17},
pmid = {1979037},
title = {{Molecular evolution of the Escherichia coli chromosome. III. Clonal frames.}},
url = {http://mutage.oxfordjournals.org/cgi/doi/10.1093/mutage/2.2.147 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1204208},
volume = {126},
year = {1990}
}
@article{Kibota1996,
abstract = {Mutations are a double-edged sword: they are the ultimate source of genetic variation upon which evolution depends, yet most mutations affecting fitness (viability and reproductive success) appear to be harmful. Deleterious mutations of small effect can escape natural selection, and should accumulate in small population. Reduced fitness from deleterious-mutation accumulation may be important in the evolution of sex, mate choice, and diploid life-cycles, and in the extinction of small populations. Few empirical data exist, however. Minimum estimates of the genomic deleterious-mutation rate for viability in Drosophila melanogaster are surprisingly high, leading to the conjecture that the rate for total fitness could exceed 1.0 mutation per individual per generation. Here we use Escherichia coli to provide an estimate of the genomic deleterious-mutation rate for total fitness in a microbe. We estimate that the per-microbe rate of deleterious mutations is in excess of 0.0002.},
annote = {upper bound on selection coefficient 0.012
this is in contrast to the finding in natural populations, in which they found the selection coefficient of SUBTITUTIONS to be much lower.
        
lower bound to mutation rate is 1.9e-4 per genome per generation
        
50 lines, 25 generations per bottleneck, 300 bottlenecks => 7500 generations. samples taken from bottlenecks 0 100 120 200 250 300
        
        
        
      },
author = {Kibota, Travis T. and Lynch, Michael},
doi = {10.1038/381694a0},
file = {:D$\backslash$:/papers/Kibota, Lynch - 1996 - Estimate of the genomic mutation rate deleterious to overall fitness in E. coli.pdf:pdf},
issn = {0028-0836},
journal = {Nature},
keywords = {Animals,Drosophila melanogaster,Drosophila melanogaster: genetics,Escherichia coli,Escherichia coli: genetics,Escherichia coli: growth \& development,Genetic,Mutation,Selection,bacteria,escherichia coli,experimental evolution,mutation accumulation,mutation rate},
mendeley-tags = {bacteria,escherichia coli,experimental evolution,mutation accumulation,mutation rate},
month = jun,
number = {6584},
pages = {694--6},
pmid = {8649513},
title = {{Estimate of the genomic mutation rate deleterious to overall fitness in E. coli.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/8649513},
volume = {381},
year = {1996}
}
@article{Tenaillon2000,
abstract = {Bacterial mutation rates can increase and produce genetic novelty, as shown by in vitro and in silico experiments. Despite the cost due to a heavy deleterious mutation load, mutator alleles, which increase the mutation rate, can spread in asexual populations during adaptation because they remain associated with the rare favorable mutations they generate. This indirect selection for a genetic system generating diversity (second-order selection) is expected to be highly sensitive to changes in the dynamics of adaptation. Here we show by a simulation approach that even rare genetic exchanges, such as bacterial conjugation or transformation, can dramatically reduce the selection of mutators. Moreover, drift or competition between the processes of mutation and recombination in the course of adaptation reveal how second-order selection is unable to optimize the rate of generation of novelty.},
author = {Tenaillon, Olivier and {Le Nagard}, Herve and Godelle, Bernard and Taddei, Fran\c{c}ois},
doi = {10.1073/pnas.180063397},
file = {:D$\backslash$:/papers/Tenaillon et al. - 2000 - Mutators and sex in bacteria conflict between adaptive strategies.pdf:pdf},
issn = {0027-8424},
journal = {Proceedings of the National Academy of Sciences of the United States of America},
keywords = {Adaptation,Bacteria,Bacteria: genetics,Bacterial,Bacterial Physiological Phenomena,Genome,Mutation,Physiological,bacteria,mutation hotspots,mutation rate,recombination,second order selection,simulation},
mendeley-tags = {bacteria,mutation hotspots,mutation rate,recombination,second order selection,simulation},
month = sep,
number = {19},
pages = {10465--70},
pmid = {10973474},
publisher = {National Acad Sciences},
title = {{Mutators and sex in bacteria: conflict between adaptive strategies.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/10973474},
volume = {97},
year = {2000}
}
@article{Sloan2010,
abstract = {Evolutionary theory suggests that low mutation rates should favor the persistence of asexuals. Additionally, given the observation that most nonneutral mutations are deleterious, asexuality may strengthen selection for reduced mutation rates. This reciprocal relationship raises the possibility of a positive feedback loop between sex and mutation rate. We explored the consequences of this evolutionary feedback with an individual-based model in which a sexual population is continually challenged by the introduction of asexual clones. We found that asexuals were more likely to spread in a population when mutation rates were able to evolve relative to a model in which mutation rates were held constant. In fact, under evolving mutation rates, asexuals were able to spread to fixation even when sexuals faced no cost of sex whatsoever. The added success of asexuals was the result of their ability to evolve lower mutation rates and thereby slow the process of mutation accumulation that otherwise limited their spread. Given the existence of ample mutation rate variation in natural populations, our findings show that the evolutionary feedback between sex and mutation rate may intensify the "paradox of sex," supporting the argument that deleterious mutation accumulation alone is likely insufficient to overcome the reproductive advantage of asexual competitors in the short term.},
annote = {-asexuals invading sexuals
-with cost of fidelity
-no beneficial mutations
experiments:
1. evolution of mutation rate in sexual/asexual population
2. invasion of asexuals to sexuals by mutation at the reproductive mode locus. after an MSB. 2.b invasion without evolution of mutation rate - mut rate constant at the MSB value
invasions - 30 replicates, 1000 generations.
        
results:
1. sexuals evolve higher mutation rates, and the dynamics match Dawson's theory
2. if asexual mutation rate evolved it greatly increaed their success. this is because their optimal fitness is with a lower mutation rate (1) and the starting point is the sexuals optimal mutation rate
        
problems:
this is true for a MSB. if adaptation is required, then mutator alleles will be selected for much more effectively in asexuals do to linkage disequilibrium (drives mutation rate up in asexuals) and to fisher muller effect (recombination and mutation are competing adaptive strategies).},
author = {Sloan, Daniel B. and Panjeti, Vijay G.},
doi = {10.1111/j.1558-5646.2009.00869.x},
file = {:D$\backslash$:/papers/Sloan, Panjeti - 2010 - Evolutionary feedbacks between reproductive mode and mutation rate exacerbate the paradox of sex.pdf:pdf},
issn = {1558-5646},
journal = {Evolution},
keywords = {Asexual,Computer Simulation,Evolution,Feedback,Genetic,Models,Molecular,Mutation,Reproduction,evolution of sex,mutation rate,recombination},
mendeley-tags = {evolution of sex,mutation rate,recombination},
month = apr,
number = {4},
pages = {1129--35},
pmid = {19863589},
title = {{Evolutionary feedbacks between reproductive mode and mutation rate exacerbate the paradox of sex.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/19863589},
volume = {64},
year = {2010}
}
@article{Drake1991,
abstract = {In terms of evolution and fitness, the most significant spontaneous mutation rate is likely to be that for the entire genome (or its nonfrivolous fraction). Information is now available to calculate this rate for several DNA-based haploid microbes, including bacteriophages with single- or double-stranded DNA, a bacterium, a yeast, and a filamentous fungus. Their genome sizes vary by approximately 6500-fold. Their average mutation rates per base pair vary by approximately 16,000-fold, whereas their mutation rates per genome vary by only approximately 2.5-fold, apparently randomly, around a mean value of 0.0033 per DNA replication. The average mutation rate per base pair is inversely proportional to genome size. Therefore, a nearly invariant microbial mutation rate appears to have evolved. Because this rate is uniform in such diverse organisms, it is likely to be determined by deep general forces, perhaps by a balance between the usually deleterious effects of mutation and the physiological costs of further reducing mutation rates.},
annote = {mu=0.0033},
author = {Drake, John W.},
doi = {10.1073/pnas.88.16.7160},
file = {:D$\backslash$:/papers/Drake - 1991 - A constant rate of spontaneous mutation in DNA-based microbes.pdf:pdf},
issn = {0027-8424},
journal = {Proceedings of the National Academy of Sciences of the United States of America},
keywords = {Bacterial,Bacterial: genetics,Bacteriophage lambda,Bacteriophage lambda: genetics,Base Composition,Biological Evolution,Coliphages,Coliphages: genetics,DNA,Escherichia coli,Escherichia coli: genetics,Fungal,Fungal: genetics,Genes,Mutation,Neurospora crassa,Neurospora crassa: genetics,Saccharomyces cerevisiae,Saccharomyces cerevisiae: genetics,Species Specificity,T-Phages,T-Phages: genetics,Viral,Viral: genetics,mutation rate},
mendeley-tags = {mutation rate},
month = aug,
number = {16},
pages = {7160--4},
pmid = {1831267},
title = {{A constant rate of spontaneous mutation in DNA-based microbes.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=52253},
volume = {88},
year = {1991}
}
@article{Hill1966,
abstract = {(i) A computer simulation study has been made of selection on two linked loci in small populations, where both loci were assumed to have additive effects on the character under selection with no interaction between loci. If N is the effective population size, i the intensity of selection in standard units, $\alpha$ and $\beta$ measure the effects of the two loci on the character under selection as a proportion of the pheno-typic standard deviation and c is the crossover distance between them, it was shown that the selection process can be completely specified by Ni$\alpha$, Ni$\beta$and Nc and the initial gene frequencies and linkage disequilibrium coefficient. It is then easily possible to generalize from computer runs at only one population size. All computer runs assumed an initial population at linkage equilibrium between the two loci. Analysis of the results was greatly simplified by considering the influence of segregation at the second locus on the chance of fixation at the first (defined as the proportion of replicate lines in which the favoured allele was eventually fixed). (ii) The effects of linkage are sufficiently described by Nc. The relationship between chance of fixation at the limit and linkage distance (expressed as 2Nc/(2Nc + 1)) was linear in the majority of computer runs. (iii) When gene frequency changes under independent segregation were small, linkage had no effect on the advance under selection. In general, segregation at the second locus had no detectable influence on the chance of fixation at the first if the gene effects at the second were less than one-half those at the first. With larger gene effects at the second locus, the chance of fixation passed through a minimum and then rose again. For two loci to have a mutual influence on one another, their effects on the character under selection should not differ by a factor of more than two. (iv) Under conditions of suitable relative gene effects, the influence of segregation at the second locus was very dependent on the initial frequency of the desirable allele. The chance of fixation at the first, plotted against initial frequency of the desirable allele at the second, passed through a minimum when the chance of fixation at the second locus was about 08. (v) A transformation was found which made the influence of segregation at the second locus on the chance of fixation at the first almost independent of initial gene frequency at the first and of gene effects at the first locus when these are small. (vi) In the population of gametes at final fixation, linkage was not at equilibrium and there was an excess of repulsion gametes. (vii) The results were extended to a consideration of the effect of linkage on the limits under artificial selection. Linkage proved only to be of importance when the two loci had roughly equal effects on the character under selection. The maximum effect on the advance under selection occurred when the chance of fixation at both of the loci was between 07 and 08. When the advance under selection is most sensitive to changes in recombination value, a doubling of the latter in no case increased the advance under selection by more than about 6\%. The proportion selected to give maximum advance under individual selection (05 under independent segregation) was increased, but only very slightly, when linkage is important. (viii) These phenomena could be satisfactorily accounted for in terms of the time scale of the selection process and the effective size of the population within which changes of gene frequency at the locus with smaller effect must take place.},
author = {Hill, W G and Robertson, Alan},
doi = {10.1017/S0016672300010156},
issn = {0016-6723},
journal = {Genetical research},
keywords = {Chromosome Mapping,Operations Research,hill-robertson,recombination},
mendeley-tags = {hill-robertson,recombination},
month = dec,
number = {3},
pages = {269--94},
pmid = {5980116},
title = {{The effect of linkage on limits to artificial selection.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/5980116 http://www.journals.cambridge.org/abstract\_S0016672300010156},
volume = {8},
year = {1966}
}


</div>


<div id="disqus_thread"></div>
<script type="text/javascript">

	/* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
	/* see http://help.disqus.com/customer/portal/articles/472098-javascript-configuration-variables */
	var disqus_shortname = 'yoavram'; // required: replace example with your forum shortname
	var disqus_identifier = 'tenaillon-et-al-2000';
	
	/* * * DON'T EDIT BELOW THIS LINE * * */
	(function() {
		var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
		dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
		(document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
<a href="http://disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>





		</div>
		<div class="span3">
			<div class="sidebar">
				

		
			</div>
		</div>

	</div>

	<!-- FOOTER -->
	<div class="row-fluid">
		<div class="span8 offset2">
			<footer class="colophon"> 
				
				
				<ul class="quick-links">
					<li> 
						The content on "<span xmlns:dct="http://purl.org/dc/terms/" href="http://purl.org/dc/dcmitype/Text" property="dct:title" rel="dct:type">The Mutation-Selection Blog</span>" by <a xmlns:cc="http://creativecommons.org/ns#" href="http://www.yoavram.com/" property="cc:attributionName" rel="cc:attributionURL">Yoav Ram</a> is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/">CC BY-SA 3.0</a> license. The rest is licensed separately:
					</li>
				</ul>
				<ul class="quick-links">
					<li>Front</li>
					<li>
						<a href="http://blog.getbootstrap.com/">Twitter Bootstrap</a>
					</li>
					<li>
						<a href="http://glyphicons.com/" title="CC BY-SA 3.0">Glyphicons Free</a>
					</li>
					<li>
						<a href="http://code.google.com/p/google-code-prettify/">Google Code Prettifier</a>
					</li>
					<li>
						<a href="http://ethanschoonover.com/solarized">Solarized</a>
					</li>
					<li>
						<a href="http://mathjax.org/">MathJax</a>
					</li>
				</ul>

				<ul class="quick-links">
					<li>Back</li>
					<li><a href="http://python.org/">Python</a></li>
					<li><a href="http://flask.pocoo.org/">Flask</a></li>
					<li><a href="http://pages.github.com/">GitHub Pages</a></li>	
					<li><a href="http://daringfireball.net/projects/markdown/">Markdown</a></li>
					<li><a href="http://johnmacfarlane.net/pandoc">Pandoc</a></li>
				</ul>

				<ul class="quick-links">
					<li>Code</li>
					<li><a href="https://github.com/yoavram/yoavram.github.com">Static site</a></li>
					<li><a href="https://github.com/yoavram/yoavram.github.com/tree/source">Site generator</a></li>
					<li>Logo</li>
					<li><a href="http://www.idancohen.com/">Idan Cohen</a></li>
					<li><a href="http://www.dafont.com/sketch-block.font">Sketch Block</a></li>
					<li><a href="http://www.dafont.com/positiv-a.font">Positiv-A</a></li>
				</ul>

				<ul class="quick-links">
					<li>This page was last modified on 20 February 2013 at 17:11.</li>
				</ul>

				
				<!-- Google Analytics www.google.com/analytics/ -->
				<script type="text/javascript">

				var _gaq = _gaq || [];
				_gaq.push(['_setAccount', 'UA-3865698-10']);
				_gaq.push(['_trackPageview']);

				(function() {
					var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
					ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
					var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
				})();
				</script>
				
				<a href="/recent.atom" rel="alternate" title="Recent Articles" type="application/atom+xml"></a>
			</footer>
		</div>
	</div>

</body>
</html>